samza-core/src/main/java/org/apache/samza/container/RunLoopFactory.java

```diff
@@ -39,9 +41,10 @@ public static Runnable createRunLoop(scala.collection.immutable.Map<TaskName, Ru
       ExecutorService threadPool,
       long maxThrottlingDelayMs,
       SamzaContainerMetrics containerMetrics,
-      TaskConfig taskConfig,
+      Config config,
```

Comment:
Preferable to pass elastcitrity factor as the additional variable in addition to task-config, rather than the entire config object